compiler/src/dotty/tools/dotc/cc/CaptureRef.scala

```diff
@@ -93,12 +93,19 @@ trait CaptureRef extends TypeProxy, ValueType:
   final def invalidateCaches() =
     myCaptureSetRunId = NoRunId
 
-  /** x subsumes x
-   *   this subsumes this.f
```

Comment:
Why remove this rule?

Reply:
I'm still updating the comments. This should be:

```
x subsumes y ==> x subsumes y.f
```